### PR TITLE
Feature/2 project dependend configs

### PR DESCRIPTION
**Pull request by namjul**
---

Issue: #2 

---

After merge create release tag 1.2.0

**how to test**
- test configs in respective repositories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,26 @@ Use it by setting the extends field in your `.eslintrc`:
...
+  extends: '@pmedianetwork/eslint-config/insights',
+};
+```
+
+## For datatab 
 
 ```js
 {
-  extends: '@pmedianetwork/eslint-config/prettier',
+  extends: '@pmedianetwork/eslint-config/datatab',
```
Comment (danielbartsch) [inline README.md]: datatap

**Review by danielbartsch — CHANGES_REQUESTED** [review]

**Review by TkDodo — APPROVED** [review]

**Review by georggrubner — APPROVED** [review]

**Review by schneckentempo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (namjul) [reply README.md]: ups

Comment (namjul) [reply README.md]: https://github.com/pmedianetwork/eslint-config/pull/3/commits/2be5625f996ff0373ad2a08ad29b4a72a2323eed

Comment (danielbartsch) [reply README.md]: the comment above it also has this typo

Comment (namjul) [reply README.md]: https://github.com/pmedianetwork/eslint-config/pull/3/commits/a6e871e7ffacf54e397a2b97d59e30b9fdc0012c

**Review by danielbartsch — APPROVED** [review]

**Comment by schneckentempo** [comment]
### Test Run 001

:red_circle: on adverity repo using this branch, 173 errors are produced by eslint, using 1.0.0 produces 8, 1.1.1 on the other hand produces 137 errors
:red_circle: datatap now runs at least a valid config. But there are errors and warnings for flowtype rules. Guess they come from plugin `flowtype/recommended`
image

**Comment by namjul** [comment]
Yea of course. `flowtype/recommended` should be in the default config.

https://github.com/pmedianetwork/eslint-config/pull/3/commits/be89530afef49f14d6ef3ca6a263fd04988956d2

**Comment by schneckentempo** [comment]
### Test Run 002

:heavy_check_mark: for datatap no flowtype errors are shown anymore
:heavy_check_mark: adverity still needs to use version 1.0.0 and a followup issue for 1.2.0 later https://github.com/pmedianetwork/adverity/issues/9177

**Comment by vlabs** [comment]
> or datatab no ...

datata**p**

**Pull request merged**